posts/2019-09-06-como-um-patinho-me-ajudou-a-ser-um-programador-melhor.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+date: '2019-09-06 10:06:02'
+image: /assets/img/rubber-duck.jpg
+title: Como um patinho me ajudou a ser um programador melhor
+description: >-
+  Sim, é isso mesmo, um patinho de borracha pode ajudar demais a você encontrar
+  seus bugs.
+introduction: >-
+  Sim, é isso mesmo, um patinho de borracha pode ajudar demais a você encontrar
+  seus bugs.
+twitter_text: >-
+  Sim, é isso mesmo, um patinho de borracha pode ajudar demais a você encontrar
+  seus bugs.
+main-class: dev
+color: '#637a91'
+tags:
+  - debug
+  - code
+---
+## Introdução
+
+Olá pessoal, eu sei que esse título parece estranho demais e não faz sentido algum, mas prometo que se você ler até o final, vai ver que é tudo verdade! Vai ser um post bem pequeno, mas super efetivo.
+
+Hoje eu estou escrevendo esse post de Porto de Galinhas, que é um paraíso absurdamente lindo! Estou num encontro da minha empresa ([Toptal](https://www.toptal.com/)), e é incrível poder escrever esse post com uma vista tão linda. Abaixo uma foto que tirei do meu drone:
+
+![Vista da praia e da cidade de um drone.](/assets/img/porto-de-galinhas.jpeg)
+
+Enquanto escrevo, vou ouvindo o [novo album da banda Tool](https://open.spotify.com/album/7acEciVtnuTzmwKptkjth5?si=P4isRw-xSuOcVS1F26rPJA) que está incrível.
+
+Bom, já falei demais, vamos ao que importa!
+
+## Qual é desse pato afinal?
+
+A primeira vez que eu ouvi sobre essa técnica foi no meu livro favorito de programação ~~se você achava que era o Clean Code, erroooou~~, que é o [The Pragmatic Programmer](https://www.amazon.com.br/Pragmatic-Programmer-Journeyman-Master/dp/020161622X), inclusive já escrevi um post com [65 dicas para ser um programador melhor](https://willianjusten.com.br/65-dicas-para-ser-um-programador-melhor/) que foi também retirado do livro.
```

Comment:
A primeira vez que eu ouvi sobre essa técnica foi no meu livro favorito de programação, ~~se você achava que era o Clean Code, erroooou~~, ~que~ é o [The Pragmatic Programmer](https://www.amazon.com.br/Pragmatic-Programmer-Journeyman-Master/dp/020161622X), inclusive já escrevi um post com [65 dicas para ser um programador melhor](https://willianjusten.com.br/65-dicas-para-ser-um-programador-melhor/) que ~foi também~ `foram` retirad~o~ `(as)` do livro.

Reply:
**Nota:** Mantenha em riscado a parte do (se você achava que era o Clean Code, erroooou), mas entre as vírgulas.